feat: LEAP-1833: Pause limits settings for project

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13462028230)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/421
<!-- Follow Merge -->

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13393170136

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13394605691

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1833'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13440223213

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13440223213

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1833'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13457967370

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13457967370

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13458026230)
> Successfully merged: Already up to date.

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13458351158

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1833'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13460246286

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13460246286

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13460246286

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1833'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13461914911

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13461914911

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1833'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13462028230

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13462028230
